sql/core/src/main/scala/org/apache/spark/sql/execution/exchange/BroadcastExchangeExec.scala

```diff
@@ -179,8 +179,9 @@ case class BroadcastExchangeExec(
             // SparkFatalException, which is a subclass of Exception. ThreadUtils.awaitResult
             // will catch this exception and re-throw the wrapped fatal throwable.
             case oe: OutOfMemoryError =>
+              val tables = child.collect { case f: FileSourceScanExec => f.tableIdentifier }.flatten
```

Comment:
So we can't give similar suggestions for the V2 table? Need more refactor work?

Reply:
Does we fully support analyze v2 tables? For example: analyze delta tables?

Reply:
no, ANALYZE TABLE only works for file source tables I think. And file source v2 is not production-ready.

Reply:
Got it